src/main_app/routes.js

```diff
@@ -17,6 +18,9 @@ function Routes () {
       <Route path='/privacy-policy'>
         <PrivacyPolicy />
       </Route>
+      <Route path='/about-us'>
```

Comment:
Probemos esto junto con los otros dos comentarios que te dejo. El problema como lo hiciste vos es que cuando abrís el modal de contacto, se cambia a la página de la Home. Cuando lo cerrás, se queda en la Home.
```suggestion
      <Route path='/about-us(/contact)?'>
```